### ISSUE TITLE
"X-Refs from" should display Hexdump preview for DATA type reference

### ISSUE THREAD

**Issue by elikaski** [radareorg/cutter#1707]
**Is your feature request related to a problem? Please describe.**
Under X-Refs widget, DATA type reference are displayed in code preview
It is usually meaningless to interpret data as code, while it is better to show data as Hexdump

![image](https://user-images.githubusercontent.com/37732584/62831861-0af72f80-bc2e-11e9-9d02-6882800adadc.png)

**Describe the solution you'd like**
1. There should be an option to display in Hexdump preview
2. Cutter will show Hexdump preview for DATA reference by default

**Describe alternatives you've considered**

**Additional context**

**Comment by ITAYC0HEN** [comment]
Good idea! @optizone8 you might want to have a look at it

**Comment by karliss** [comment]
In theory  disassembly output takes into account data type and displays it accordingly, unfortunately analysis isn't good enough. R2 should probably mark non executable sections as data by default.
![a1](https://user-images.githubusercontent.com/7101031/62832918-cf189600-bc3e-11e9-9e45-89017f4a2ec4.png)
![a2](https://user-images.githubusercontent.com/7101031/62832920-d50e7700-bc3e-11e9-9e0b-9b2d92f4df1c.png)
![a3](https://user-images.githubusercontent.com/7101031/62832921-d93a9480-bc3e-11e9-9d18-5c97a8a43ca5.png)

**Comment by ITAYC0HEN** [comment]
In these examples, were you the one who marked these addresses as "string", "qword", etc?
Well, in general, I think that having a hexdump preview for DATA is something we can decide and do ourselves. With or without radare2 since we have the knowledge that it is DATA ref.

**Comment by karliss** [comment]
In the examples I above no manual marking was applied. It was automatically detected during analysis.

**Comment by karliss** [comment]
Cutter should probably not display Code for "X-Refs from" with type data.

**Comment by ITAYC0HEN** [comment]
If I have an instruction like this `lea ebx [0xadd7e55]`, I would be happy to look at x-refs widget and see that in "[0xadd7e55]" there is actually a string that surrounded by more interesting strings

**Comment by xarkes** [comment]
Yes, but r2 is supposed to show that @ITAYC0HEN, see below screenshot:
![image](https://user-images.githubusercontent.com/10772605/65635518-c3abdf00-dfe0-11e9-8595-77f44efd1720.png)

**Comment by ITAYC0HEN** [comment]
look at the screenshot posted by @elikaski, this is a great example
